crates/api_models/src/events/payment.rs

```diff
@@ -10,28 +10,34 @@ use super::{
     not(feature = "payment_methods_v2")
 ))]
 use crate::payment_methods::CustomerPaymentMethodsListResponse;
-#[cfg(feature = "v1")]
-use crate::payments::{PaymentListFilterConstraints, PaymentListResponseV2};
 #[cfg(all(feature = "v2", feature = "payment_methods_v2"))]
-use crate::{events, payment_methods::CustomerPaymentMethodsListResponse};
+use crate::payment_methods::CustomerPaymentMethodsListResponse;
```

Comment:
For v2, please don't import the Structs/Traits directly. Instead import it like this:

`use crate::payment_methods`;

and use it like:

`payment_methods::CustomerPaymentMethodsListResponse`

Since we are refactoring imports, its a good opportunity to improve this situation. But only do this for v2 imports at the moment.

This also applies to the other PRs solving warnings